jobs/master/spec

```diff
@@ -32,4 +32,8 @@ properties:
 
   bind9.options:
     description: "Map of additional options"
-    default: {}
\ No newline at end of file
+    default: {}
+
+  bind9.forward.zones:
+    description: "Forward zones in forward only mode,e.g. {name: zone1, forwarders: [1.1.1.1, 1.1.1.2]}"
```

Comment:
The spec description seems to indicate that I don't have to generate literal BIND9 configurations for the forwarding zones, but the implementation seems to differ markedly.